amazon-kinesis-client/src/test/java/software/amazon/kinesis/utils/TestRecordProcessor.java

```diff
@@ -0,0 +1,104 @@
+package software.amazon.kinesis.utils;
+
+import lombok.extern.slf4j.Slf4j;
+import org.slf4j.MDC;
+import software.amazon.kinesis.exceptions.InvalidStateException;
+import software.amazon.kinesis.exceptions.ShutdownException;
+import software.amazon.kinesis.lifecycle.events.LeaseLostInput;
+import software.amazon.kinesis.lifecycle.events.ShardEndedInput;
+import software.amazon.kinesis.lifecycle.events.ShutdownRequestedInput;
+import software.amazon.kinesis.processor.ShardRecordProcessor;
+import software.amazon.kinesis.lifecycle.events.InitializationInput;
+import software.amazon.kinesis.retrieval.KinesisClientRecord;
+
+import java.nio.ByteBuffer;
+
+@Slf4j
+public class TestRecordProcessor implements ShardRecordProcessor {
+
+    private static final String SHARD_ID_MDC_KEY = "ShardId";
+
+    private String shardId;
+
+    RecordValidatorQueue recordValidator;
+
+    public TestRecordProcessor(RecordValidatorQueue recordValidator) {
+        this.recordValidator = recordValidator;
+    }
+
+    @Override
+    public void initialize(InitializationInput initializationInput) {
+        shardId = initializationInput.shardId();
+        MDC.put(SHARD_ID_MDC_KEY, shardId);
+        try {
+            log.info("Initializing @ Sequence: {}", initializationInput.extendedSequenceNumber());
+        } finally {
+            MDC.remove(SHARD_ID_MDC_KEY);
+        }
+    }
+
+
+    @Override
+    public void processRecords(software.amazon.kinesis.lifecycle.events.ProcessRecordsInput processRecordsInput) {
```

Comment:
Why didn't you import this package just like others `mport software.amazon.kinesis.lifecycle.events.ShutdownRequestedInput;`